.github/workflows/create-reports.yaml

```diff
@@ -141,7 +146,8 @@ jobs:
           ${{ needs.read_inputs.outputs.tools_table }}
           ## All evaluated models
           ${{ needs.read_inputs.outputs.models_table }}
-        assignees: ${{ github.actor }}
         reviewers: ${{ github.actor }}

```

Comment:
it's enough to have the reviewer, no need to assign the PR.
And I get only half of those emails then :)